doc/rtd/spelling_word_list.txt

```diff
@@ -0,0 +1,294 @@
+akamai
```

Comment:
I see a number of US spellings in this list. E.g.,artifact, honored, flavors. Shouldn't we be disallowing these words?